source_me.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+MPI_TARGET=${1}
+
+
+# Custom parameters
+
+MPICH_DIR="<path_to_mpich>" #e.g./scratch/POC/mpich
```

Comment:
Can we move these config to run_build.sh? and have run_build source this file?

Reply:
Done